spec/support/request/authentication_workflow.rb

```diff
@@ -37,6 +37,18 @@ def create_enterprise_user( attrs = {} )
     new_user
   end
 
+  def create_admin( attrs = {} )
+    new_user = build(:user, attrs)
+    new_user.spree_roles = [Spree::Role.find_or_create_by!(name: 'admin')]
+    new_user.save
+    if attrs.key? :enterprises
+      attrs[:enterprises].each do |enterprise|
+        enterprise.users << new_user
+      end
+    end
+    new_user
+  end
```

Comment:
This would work but it's not needed. I looked around to find how this should be done.
There's a :admin_user factory, you can simply do:

`allow(view).to receive_messages spree_current_user: create(:admin_user)`

I realized we should delete this create_enterprise_user method and so I created this PR that cleans up a few things related to this topic :+1: 

This test in this PR can still just use the user factory :+1: